ibis/backends/tests/sql/snapshots/test_sql/test_order_by/random/out.sql

```diff
@@ -1,5 +1,16 @@
 SELECT
   *
-FROM "star1" AS "t0"
+FROM (
+  SELECT
+    "t1"."f",
+    "t1"."c"
+  FROM (
+    SELECT
+      *
+    FROM "star1" AS "t0"
+    ORDER BY
+      RANDOM() ASC
```

Comment:
Why is there now a subquery? Is that correct?

Reply:
Did you see how the actual expression changed here? Before the expression was just `t.order_by(ibis.random())`. Now it is the more complicated `t.order_by(ibis.random()).select("f", "c").order_by(ibis.random())`.

This new version is equivalent to `t.select("f", "c").order_by(ibis.random())` which is `SELECT "t0"."f", "t0"."c" FROM "star1" AS "t0" ORDER BY RANDOM() ASC`. For this simple case we could drop the internal order by, and then merge the two selects. But my initial gut reaction is that there are cases where the internal select result depends on the internal order by, and so in general we can't just drop the inner select? So in general I would say this subquery is unnecessary, but correct.